src/global.css

```diff
@@ -18,14 +18,7 @@ body {
   margin: 0;
   padding: 0;
   text-rendering: optimizeLegibility;
-  background-image: linear-gradient(
-    120deg,
-    #eea2a2 0,
```

Comment:
You can also handle globals with styled components: https://styled-components.com/docs/api#createglobalstyle